docs/resources/Guild.md

```diff
@@ -32,7 +32,7 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | widget_enabled?               | boolean                                                                             | whether or not the server widget is enabled                                                                                      |
 | widget_channel_id?            | ?snowflake                                                                          | the channel id for the server widget                                                                                             |
 | system_channel_id             | ?snowflake                                                                          | the id of the channel to which system messages are sent                                                                          |
-| system_channel_flags          | number                                                                              | [system channel flags](#DOCS_RESOURCES_GUILD/guild-object-system-channel-flags)                                                  |
+| system_channel_flags          | integer                                                                             | [system channel flags](#DOCS_RESOURCES_GUILD/guild-object-system-channel-flags)                                                  |
 | rules_channel_id              | ?snowflake                                                                          | the id of the channel in which a discoverable server's rules should be found                                                     |
 | joined_at? \*                 | ISO8601 timestamp                                                                   | when this guild was joined at                                                                                                    |
```

Comment:
We could add an in-line link to [the standard](https://www.loc.gov/standards/datetime/iso-tc154-wg5_n0038_iso_wd_8601-1_2016-02-16.pdf) or the [w3c page on how to use it](https://www.w3.org/TR/NOTE-datetime) here, in case anyone does not know what ISO8601 is specifically.

Reply:
Should that be part of this PR too?

Reply:
only if you want, was more a random thought while reading it tbh

Reply:
I like the idea—more info never hurts!

Reply:
Done, and updated all other references of "iso8601" while at it.